slippy/tile_grid_test.go

```diff
@@ -376,139 +272,68 @@ func TestToNativeX(t *testing.T) {
 				pt = geom.Point{pts[0], pts[1]}
 			}
 
-			if !cmp.Float(pt.X(), tc.expected) {
-				t.Errorf("got %v expected %v", pt.X(), tc.expected)
+			if pt != tc.expected {
```

Comment:
You should use `cmp.PointEqual(pt, tc.expected)` (from `github.com/go-spatial/geom/cmp`). This is the package we use for floating point comparisons since, plain `==` is unstable.

Reply:
The test cases can be left as is btw

Reply:
modified as requested